pmoves/services/semantic-cache/config.py

```
@@ -0,0 +1,132 @@
+        default="http://tensorzero-gateway:3000/openai/v1",
+        description="Upstream TensorZero OpenAI-compatible base URL",
+    )
+
+    # --- pgvector (Layer 1) ---
+    database_url: str = Field(
+        default="",
+        description="PostgreSQL URL with pgvector (e.g. postgresql://user:pass@host/db)",
+    )
+    similarity_threshold: float = Field(
+        default=0.90,
+        description="Minimum cosine similarity for a cache hit (0.0–1.0)",
```

Ruff (0.15.20)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)